### DISCUSSION TITLE
`--include-unknown` is doing nothing, is this expected?

### DISCUSSION THREAD

**Discussion by peterc-s** [microsoft/winget-cli#4741] [Q&A]
Before I make an issue, is this expected behavior?

```PowerShell
PS C:\Users\peter> winget upgrade --all
No installed package found matching input criteria.
10 package(s) have version numbers that cannot be determined. Use --include-unknown to see all results.
PS C:\Users\peter> winget upgrade --include-unknown
No installed package found matching input criteria.
PS C:\Users\peter> winget upgrade --include-unknown --all
No installed package found matching input criteria.
```

**Comment by denelon** [comment]
This is usually caused when applications are installed but no version number is reported in the registry. In some cases, this is fine and WinGet can simply try to install the latest version. Unfortunately, this can lead to feeling like there are always updates available even when the latest version is installed.
